### PR TITLE
added JS API task

### PR DESCRIPTION
**Pull request by austincqdo**

### CODE REVIEW

**Comment by austincqdo** [comment]
https://bozuman.cybozu.com/g/schedule/view.csp?event=1747885&bdate=2019-07-08&uid=2304&gid=&referer_key=cb33b15242d6af7c34586de8ca67b6ef

here is a link to the app template used (in attachments).

File: js_pt1/dest/desktop/js/script5+1.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+(function() {
+	'use strict';
+	kintone.events.on('app.record.create.show', function(event) {
+		let actions = [
```
Comment (mmiyahara) [inline js_pt1/dest/desktop/js/script5+1.js]: Use `const` instead.

File: js_pt1/dest/desktop/js/script5+1.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			'Relentless quest',
+			'Mental and physical persistence',
+			'Empathy for the ideal',
+			'Inspire others',
+			'Increase knowledge and skills',
+			'Fairness and honesty'
+		]
+		let rows = [];
```
Comment (mmiyahara) [inline js_pt1/dest/desktop/js/script5+1.js]: You can use `const` in this case as the value it holds is not immutable.

> The const declaration creates a read-only reference to a value.
> It does not mean the value it holds is immutable, just that the variable identifier cannot be reassigned.
> For instance, in the case where the content is an object, this means the object's contents (e.g., its properties) can be altered.
> [const - JavaScript | MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#Description)

File: js_pt1/dest/desktop/js/script5+1.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+					'type': 'DROP_DOWN',
+					'value': action,
+				},
+				'status': {
+					'type': 'CHECK_BOX',
+					'value': ['Not reviewed']
+				},
+				'task': {
+					'type': 'MULTI_LINE_TEXT',
+					'value': undefined
```
Comment (mmiyahara) [inline js_pt1/dest/desktop/js/script5+1.js]: The value of the `'value'` should be `''` as the value of the MULTI_LINE_TEXT field is `''` when it's empty.
Please check the value of MULTI_LINE_TEXT field when it's empty.

**Review by mmiyahara — CHANGES_REQUESTED** [review]
I put some comments. Please check!

**Review by mmiyahara — APPROVED** [review]
Great! Please merge it into the `mater` branch.

**Pull request merged**